config/prod/test-scrna-large.yaml

```diff
@@ -0,0 +1,28 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v6.yaml
```

Comment:
there is a new version of the template.  please use it: https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/3